Dockerfile

```diff
@@ -40,11 +40,15 @@ ENV NODE_ENV=production
 
 WORKDIR /app
 
-COPY --from=app_builder /src/ui/node_modules ./node_modules
-COPY --from=app_builder /src/ui/package*.json ./
 COPY --from=app_builder /src/ui/next.config.js ./
+COPY --from=app_builder /src/ui/package*.json ./
 COPY --from=app_builder /src/ui/public ./public
-COPY --from=app_builder --chown=nextjs:nodejs /src/ui/.next ./.next
+COPY --from=app_builder /src/ui/.next/standalone .
+COPY --from=app_builder /src/ui/.next/static ./.next/static
+# Copy standing data package to Core repository as core dependency
+COPY --from=app_builder \
```

Comment:
Just to leave a more detailed comment - the NextJS standalone build analyses imports and it determines that these deps aren't required and deletes them.

https://github.com/ministryofjustice/bichard7-next-core/blob/13c008fd3d919c705fb851ce02716854d7f77074/src/lib/requireStandingData.ts